src/docs/product/integrations/github/index.mdx

```diff
@@ -220,6 +220,30 @@ You can also resolve issues with pull requests by including `fixes <SENTRY-SHORT
 
 When Sentry sees this, we’ll automatically annotate the matching issue with a reference to the commit or pull request, and, later, when that commit or pull request is part of a release, we’ll mark the issue as resolved.
 
+### Stack Trace Linking
+Stack trace linking takes you from a file in your Sentry stack trace to that same file in your source code. If you have commit tracking set up in Sentry, we can take you to the exact version (using the commit associated with the event) of the source code when the error occurred. Otherwise we'll link you to the current state of the source code (using the `default_branch`).
```

Comment:
@MeredithAnya hmmm...I don't know if just because you have commit tracking you'll get the exact version. You also have to tag your events with the correct release.

Reply:
@scefali I guess I figured if you went through the 'commit tracking' setup process in the docs, you should have that set up. maybe I can link 'commit tracking' to somewhere in the docs so people know what they need?